packages/angular_devkit/build_angular/src/utils/webpack-browser-config.ts

```diff
@@ -89,6 +94,8 @@ export async function generateWebpackConfig(
         esVersionInFileName: true,
```

Comment:
I think `es5BrowserSupport: undefined` should be included

Reply:
It is?

https://github.com/angular/angular-cli/pull/15246/files#diff-602ee11a21c4632098aa7846c328360cR98